### ISSUE TITLE
Framework selector menu should show the friendly name of the framework

### ISSUE THREAD

**Issue by BillHiebert** [dotnet/roslyn-project-system#1674]
1. Create a new console application
2. Cross-target by change <Framework> to <Frameworks> and specify a 2nd framework like net452. Save
3. The debug dropdown will list the current active framework and a menu to change the selection

The menu shows the shot names. Instead it should show the friendly name

**Comment by abpiskunov** [comment]
I have a pending change that contains a component that can provide friendly target frameworks names obtained from Nuget API. Will let you know when i check it in.

**Comment by davkean** [comment]
I disagree that this should be done by the project system. This should be calculated during the build, for use both in the build, and in VS.

**Comment by abpiskunov** [comment]
Build is too slow - what would you show in VS until build is not happened? We have 3 places now: editor top combobox, F5 framework selector and Solution explorer tree - all of them need friendly framework names and should be displayed when project just opened. If we wait for build and then update UI - too many flicks and bad UI design. I already have a component that talks to Nuget API and uses same Nuget assemblies build would use to determine frameworks , why not use it if we can?

**Comment by davkean** [comment]
By _build_ here, I mean during evaluation in the same way that we calculate the full TFM.
